### DISCUSSION TITLE
rAudio 20240719: playlist can not be saved and trackinfo not showing up

### DISCUSSION THREAD

**Discussion by Fixu69** [rern/rAudio#1716] [Fixed Issues]
raudio 20240719: the playlist can no longer be saved and track info is not displayed. When selecting the (+) or trackinfo, the background is darkened and nothing works anymore until refreshing the hole page with F5.

**Comment by Fixu69** [comment]
Am I the only one with this problem?

**Reply by Wheatleybloke** [reply]
Similar issue plus I noticed that after shuffling first 100 or so tracks out of 175, the rest are sorted by album, not title

**Comment by Fixu69** [comment]
For me this issue was fixed with version 20240906
